### PR TITLE
docs fixes

### PR DESCRIPTION
**Pull request by Jewcub**

### CODE REVIEW

File: docs/hub/buckets.md [hunk 1]
```diff
@@ -56,10 +63,6 @@ Buckets are dynamic folders distributed over IPFS using ThreadsDB. Each Bucket h
 
 Buckets can't change the speed that IPNS propogates through the network, but we recommend you explore and try for yourself. The [Hub gateway](#explore-on-the-gateway) will always render the latest data right away.
 
-## Buckets CLI
```
Comment (Jewcub) [inline docs/hub/buckets.md:59]: This was redundant. Same as at the top of the page

File: docs/hub/cli/tt_bucket_pull.md [hunk 2]
```diff
@@ -1,39 +1,85 @@
...
+│   one.txt
+│
+└───bar
+    │   two.txt
+    │
+    └───baz
+        │   three.txt
+```
 
-'tt bucket pull foo mydir':
```
Comment (Jewcub) [inline docs/hub/cli/tt_bucket_pull.md:12]: this multiline tabbed format was not showing up in the browser and was really hard to read.

File: docs/hub/cli/tt_bucket_push.md [hunk 3]
```diff
@@ -1,40 +1,92 @@
...
+foo
+│   one.txt
+│
+└───bar
+    │   two.txt
+    │
+    └───baz
+        │   three.txt
+```
+
```
Comment (andrewxhill) [inline docs/hub/cli/tt_bucket_push.md:22]: lovely

**Comment by andrewxhill** [comment]
merging, thanks @Jewcub!

**Pull request merged**